COMMPLAN.md

```diff
@@ -0,0 +1,204 @@
+work).
+
+#### What should we use it for?
+
+The Xdev Twitter account should be for making *announcements* to the
+general public, such as making an announcement of a new blog post.
+In general, Twitter *could* be used for any announcement, but because
+of the limitations of the platform (i.e., limiting the length of the
+tweet) it is best to tie the tweet to *another* communication platform,
+such as a blog post.
+
+#### What should we *not* use it for?
+
+The Xdev Twitter account should *not* be used for personal use.  Currently,
+only Kevin has the "keys" to the Twitter account.  If you want to make a
+broad announcement, contact Kevin via Zulip.
+
+### Email
+
+#### Why should we use it?
+
+Not sure.  I'm open to discussion on this, but I am beginning to feel like
+email (especially the `xdev@ucar.edu` address) should not be used by any
+of us, unless people explicitly reach out to us.  That is, I think email
+should be a way for people *outside* of NCAR to directly communicate with
+*us*.
+
+At NCAR, email is used substantially for inter-NCAR communication or for
+communication with specific extra-NCAR entities (like the NSF).  The amount
+of email that any one of us can easily receive is enough that it is easy to
+get lost, and so I recommend *not* using email for Team communication.
+
+#### What should we use it for?
+
+Just to reply to other people who use the `xdev@ucar.edu` address.
+
+#### What should we *not* use it for?
+
+Pretty much everything.
+
+### Other Channels
+
+There are many other communication channels for other (albeit similar)
+communities, even ones that we might consider ourselves members.  The
+most notable case is Pangeo Community, which has its own communication
+channels, as described in the [Pangeo Guide](PANGEO.md).  If directly
+interacting with an external community, such as Pangeo, use the communication
+channels *used by that community.*  Note that this could me, regretfully,
+re-posting or double-posting announcements or questions in multiple
+channels because of different community expectations.
+
+## Summary Chart
+
+Below attempts to summarize the above content into a table describing *with
+whom* each communication channel is ideally suited for communication and
+*for what kind* of communication it is best suited.
+
+| Channel | With Whom? | What Kind? |
+|---------|------------|------------|
+| GitHub  | Developers | Bug Reports, Feature Requests, Code Reviews, Design Discussions |
+| Zulip `Xdev` Stream | The Xdev Team | Questions, General Comments, Soliciting Feedback, Jokes, *etc.* |
+| Zulip Public Streams | NCAR Python Users | Announcements, Questions/Answers, Development Problems, Soliciting Feedback, *etc.* |
+| Xdev Blog | General Public | Announcements, Technical Examples, Howtos, *etc.* |
+| Twitter (`@NCARXdev`) | General Public | Announcements of Blog Posts |
+| Email (`xdev@ucar.edu`) | General Public | Incoming questions from outside entities |
```

Comment:
I too find this summary to be very useful. I am wondering if folks are okay with having this summary at the beginning of the document since it provides the reader with all necessary information without having to read the entire document ;)?

Reply:
Great idea!  Lead with the simple.

Reply:
Done.